files/default/neutron-ha-tool.py

```diff
@@ -576,6 +600,195 @@ def l3_agent_evacuate(qclient, agent_host, agent_picker, router_filter,
     return errors
 
 
+def dhcp_agent_evacuate(qclient, agent_host, agent_picker,
+                      noop=False, wait_for_network=True, fail_fast=False):
+    """
+    Retreive a list of networks scheduled on the listed DHCP agent, and move that
+    to another agent.
+
+    :param qclient: A neutronclient
+    :param noop: Optional noop flag
+    :param agent_host: the hostname of the DHCP agent to migrate networks from
+    :returns: total number of errors encountered
+
+    """
+
+    agent_list = list_agents(qclient, agent_type='DHCP agent')
+
+    agent_to_evacuate = None
+    for agent in agent_list:
+        if agent.get('host', None) == agent_host:
+            agent_to_evacuate = agent
+            break
+
+    if not agent_to_evacuate:
+        LOG.error("Could not locate agent to evacuate; aborting!")
+        return 0
+
+    target_list = list_alive_dhcp_agents_except(agent_list,
+                                                exclude_agent=agent_to_evacuate)
+
+    if len(target_list) < 1:
+        LOG.error("There are no DHCP agents alive to migrate routers onto")
```

Comment:
s/routers/networks

Reply:
done + fixed one log message in L3 migration code (separate commit)